ReactWindows/ReactNative/Views/Picker/ReactPickerShadowNode.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Facebook.CSSLayout;
+using ReactNative.UIManager;
+using Windows.UI.Xaml.Controls;
+
+namespace ReactNative.Views.Picker
+{
+    /// <summary>
+    /// The shadow node implementation for Picker views.
+    /// </summary>
+    public class ReactPickerShadowNode : LayoutShadowNode
+    {
+        /// <summary>
+        /// Instantiates the <see cref="ReactPickerShadowNode"/>.
+        /// </summary>
+        public ReactPickerShadowNode()
+        {
+            MeasureFunction = MeasurePicker;
+        }
+
+        private static MeasureOutput MeasurePicker(CSSNode node, float width, float height)
+        {
+            return new MeasureOutput(width, 40);

```

Comment:
40 is the default value if no height is given. If style has height either smaller or larger than 40 the picker will use that.